### PR TITLE
Update GCE PD CSI Driver feature list

### PR DESCRIPTION
**Pull request by saad-ali**
Fixes https://github.com/kubernetes-csi/docs/issues/310

### CODE REVIEW

File: book/src/drivers.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
...
 [DriveScale](https://github.com/DriveScale/k8s-plugins) | `csi.drivescale.com` | v1.0 |A Container Storage Interface (CSI) Driver for DriveScale software composable infrastructure solution | Persistent | Read/Write Single Pod | Yes | 
 [Ember CSI](https://ember-csi.io) | `[x].ember-csi.io` | v0.2, v0.3, v1.0 | Multi-vendor CSI plugin supporting over 80 Drivers to provide block and mount storage to Container Orchestration systems. | Persistent | Read/Write Single Pod | Yes | Raw Block, Snapshot
-[GCE Persistent Disk](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver) | `pd.csi.storage.gke.io` | v0.3, v1.0 | A Container Storage Interface (CSI) Driver for Google Compute Engine Persistent Disk (GCE PD) | Persistent | Read/Write Single Pod | Yes | Snapshot
+[GCE Persistent Disk](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver) | `pd.csi.storage.gke.io` | v0.3, v1.0 | A Container Storage Interface (CSI) Driver for Google Compute Engine Persistent Disk (GCE PD) | Persistent | Read/Write Single Pod | Yes | Snapshot, Expansion, Topology
```
Comment (msau42) [inline book/src/drivers.md:35]: also raw block

Comment (saad-ali) [reply book/src/drivers.md:35]: https://github.com/kubernetes-csi/docs/pull/313

**Pull request merged**